components/brave_ads/browser/ads_service_impl.h

```diff
@@ -190,6 +190,7 @@ class AdsServiceImpl : public AdsService,
       const std::string& json);
   void RetryViewingAdWithId(
       const std::string& id);
+  void ResetTheWholeState(const base::Callback<void(bool)>& callback) override;
```

Comment:
if this is different than `ResetAllState` then both of them need to be renamed and comments added because this is really confusing to have `ResetAllState` and `ResetTheWholeState`

Reply:
`ResetTheWholeState` has same functionality, please see `ShutdownBatAds` for usage

Reply:
NONCRITICAL
Covered by: https://github.com/brave/brave-browser/issues/6239